client/src/actions/index.js

```diff
@@ -24,23 +24,26 @@ export function getTrips(
     list = []
 ) {
     const request = axios
-        .get(`/api/getManyTrips?limit=${limit}&skip=${start}&order=${order}`)
-        .then(response => [...list, ...response.data]);
+        .get(`/api/getManyTrips?limit=${limit}&skip=${start}&order=${order}`);
 
-    return {
-        type: GET_TRIPS,
-        payload: request
-    };
+    return (dispatch) => {
```

Comment:
Looks like this is the correct way to write async actions, otherwise it is not possible to test them